src/components/ChecLoading.vue

```diff
@@ -67,6 +102,10 @@ export default {
   }
 }
 
+.secondary-animation {
+  @apply animate-spin h-full w-full;
+}
```

Comment:
I disagree with this approach. We should use the same loading component everywhere. If there are new loading components we can add them as new components - making a component like this work in two ways is an anti-pattern. You're better off making a parent component like `ChecLoading` then creating `Loading` and `FileLoading` components that extend them and using those.

All I've seen design-wise is your animated GIF in the PR description. It looks like it's a regular `ChecLoading` component but in a different colour. Is that right? If so, maybe we can add a `variant` prop and accept some sort of "inverted" option where we switch the colour?

This could also be separated into a different issue and pull request too.

Reply:
Implementation wise it appears that it's colour and also the direction the spinner moves in. My recommendation is that we ask Blaze to use a consistent loading indicator, and give it a variant that works on a dark background.

Reply:
Thank you Robbie. I'll make a separate PR introducing the loading variation in the form of components extending chec-loading. 

>All I've seen design-wise is your animated GIF in the PR description. It looks like it's a regular ChecLoading component but in a different colour. Is that right? If so, maybe we can add a variant prop and accept some sort of "inverted" option where we switch the colour?

Yes you're right, it also renders a white overlay that's not prop configurable. Should the variant prop handle that also?